### DISCUSSION TITLE
Tensorflow support for misc. pruning algorithms

### DISCUSSION THREAD

**Discussion by ilyanelken** [microsoft/nni#3799] [Ideas]
Hi,

I see that almost all the pruning algorithms are supported for PyTorch.
When there will be support for TF?

Thanks,
Ilya

**Comment by liuzhe-lz** [comment]
Currently we don't have enough devs working on TensorFlow support, so the progress will be relatively slow. I estimate it would be one algorithm per release in following months.
You can suggest the algorithm you most want to try, and we will put it on higher or maybe highest priority.
